content/doc/book/installing/_docker-for-tutorials.adoc

```diff
@@ -81,13 +142,13 @@ and Linux>> instructions above.
 
 If you have some experience with Docker and you wish or need to access the
 Jenkins/Blue Ocean Docker container through a terminal/command prompt using the
-link:https://docs.docker.com/engine/reference/commandline/exec/[`docker exec`]
+link:https://docs.docker.com/engine/reference/commandline/container_exec/[`docker container exec`]
 command, you can add an option like `--name jenkins-tutorials` (with the
-link:https://docs.docker.com/engine/reference/commandline/run/[`docker run`]
+link:https://docs.docker.com/engine/reference/commandline/container_run/[`docker container run`]
 above), which would give the Jenkins/Blue Ocean Docker container the name
 "jenkins-tutorials".
 
 This means you could access the Jenkins/Blue Ocean container (through a separate
-terminal/command prompt window) with a `docker exec` command like:
+terminal/command prompt window) with a `docker container exec` command like:
 
-`docker exec -it jenkins-tutorials bash`
+`docker container exec -it jenkins-tutorials bash`
```

Comment:
Since the earlier command is naming it `jenkins-tutorial`, use the same name here for convenience
```suggestion
`docker container exec -it jenkins-tutorial bash`
```